python/cryptography/security/insecure-hash-algorithms-md5.yaml

```diff
@@ -32,3 +31,10 @@ rules:
   severity: WARNING
   languages:
   - python
+  patterns:
+    - pattern: cryptography.hazmat.primitives.hashes.$MD5()
+    - metavariable-regex:
+        metavariable: $MD5
+        regex: ^(MD5)$
+    - focus-metavariable: $MD5
+  fix: SHA256
```

Comment:
Any reason why you recommend SHA256? We had SHA3 family since, but also the blake family which IIRC has a better SIMD utilization